core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/kubernetes/WhiskPodBuilder.scala

```diff
@@ -113,6 +121,13 @@ class WhiskPodBuilder(client: NamespacedKubernetesClient,
     pod
   }
 
+  def calculateCpu(c: KubernetesCpuScalingConfig, memory: ByteSize): Int = {
+    val cpuPerMemorySegment = c.millicpus
+    val cpuMin = c.millicpus
+    val cpuMax = c.maxMillicpus
+    math.min(math.max((memory.toMB / c.memory.toMB) * cpuPerMemorySegment, cpuMin), cpuMax).toInt
```

Comment:
@tysonnorris Just checking if we should be casting to `double` here as its an integer based division

Reply:
Currently calculation is always in millicpu in all cases - we append "m" to the pod spec cpu limit. So truncating the decimal places works. If instead of configuring 100 for the cpu per mem segment in millicpu, we could configure it as 0.1 and truncate the decimal (not sure if kubernetes cares about how many decimals are used for cpu?). Millicpu seemed more readable, but I don't have a strong opinion.